### PR TITLE
remove reliance on meta data and instead rely on POST data for lists being used.

### PR DESCRIPTION
**Pull request by tw2113**
The metadata now stores ALL of the chosen lists as meta, and not just items checked by the user.

### CODE REVIEW

File: includes/class-mail.php [hunk 1]
```diff
@@ -301,10 +301,15 @@ public function mail( $destination_email, $content, $submission_details, $was_fo
 
 		$content_title  = '<p><strong>' . esc_html__( 'Form title: ', 'constant-contact-forms' ) . '</strong>' . get_the_title( $submission_details['form_id'] ) . '<br/>';
 
-		$list_ids = get_post_meta( (int) $submission_details['form_id'], '_ctct_list', true );
-		if ( ! is_array( $list_ids ) ) {
-			$list_ids = [ $list_ids ];
+		if ( ! empty( $_POST ) && is_array( $_POST ) ) {
```
Comment (ramsesdelr) [inline includes/class-mail.php:304]: Can we add a `//phpcs:ignore` to this line and the one below to avoid the nonce  warning from PHPCS?

**Review by ramsesdelr — CHANGES_REQUESTED** [review]
Added a quick change, can you please take a look @tw2113 ?

File: includes/class-mail.php (hunk 1 above)
Comment (tw2113) [reply includes/class-mail.php]: This should be resolved in 935d431b

**Review by ramsesdelr — APPROVED** [review]
Looks good, approved, thanks!

**Pull request merged**